proto/store/plan_check_run.proto

```diff
@@ -0,0 +1,35 @@
+syntax = "proto3";
+
+package bytebase.store;
+
+option go_package = "generated-go/store";
+
+message PlanCheckRunConfig {
+  int32 sheet_id = 1;
+  int32 database_id = 2;
+}
+
+message PlanCheckRunResult {
```

Comment:
Since we will redo the plan check, I wonder if we should have better storage structure? For example, SQL review check gets its own.